packages/ace-report/src/report-builders.js

```diff
@@ -110,6 +110,12 @@ class ReportBuilder {
   build() {
     return this._json;
   }
+  // run the function on the given item under _json.data
+  cleanData(key, fn) {
```

Comment:
I'm wondering if this is worth being added as a utility to `ReportBuilder`.
Do we plan other use? Or outside the `data` section?
If we're to keep it, maybe `transformData` is a more explicit function name.

Reply:
Ace needed some way to remove `path` after the images get copied but before saving/printing output, and putting it in `ReportBuilder` is the only safe place to access `_json`. We could put all the logic there but I liked it better as as visitor-style function (albeit not very generic), because `ReportBuilder` doesn't know anything about the details of the `data` section in the first place.

Like I said, I'd rather not put temporary data into the report at all (or maybe it could be more explicitly in a `temp` section, but this works for now).

Reply:
all right, makes sense!